src/pages/recording/recording-page.ts

```diff
@@ -74,61 +72,65 @@ export class RecordingPage extends LitElement {
     `;
   }
 
-  static styles = css`
-    :host {
-      display: flex;
-      flex-direction: column;
-      align-items: center;
-      width: 100%;
-      height: 100%;
-    }
+  static styles = [
+    PAGE_STYLES,
+    css`
+      :host {
+        display: flex;
+        flex-direction: column;
+        align-items: center;
+        width: 100%;
+        height: 100%;
+        flex: 1;
```

Comment:
Is this to control how this component is rendered in the host? If so, it should move to the host.